packages/prediction-app/src/features/model-templates-overview/ModelTemplatesOverview.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import { PageHeader } from '../common-features/PageHeader/PageHeader'
+import { useModelTemplates } from '../../hooks/useModelTemplates'
+import { useRoute } from '../../hooks/useRoute'
+import { Button, CircularLoader, IconArrowLeft16 } from '@dhis2/ui'
+import i18n from '@dhis2/d2-i18n'
+import styles from './ModelTemplatesOverview.module.css'
+import ModelTemplateConfigForm, { ModelTemplateConfigFormValues } from './components/ModelTemplateConfigForm/ModelTemplateConfigForm'
+import { useConfigureModelTemplate } from './hooks/useConfigureModelTemplate'
+import { useNavigate } from 'react-router-dom'
+
+export const ModelTemplatesOverview = () => {
+    const { route } = useRoute()
+    const { modelTemplates, error, isLoading: isLoadingTemplates } = useModelTemplates({ route })
+    const { mutate: configureModelTemplate, isLoading: isSubmitting } = useConfigureModelTemplate()
```

Comment:
Can you refactor this to be only `useConfiguredModels`? Also change the name of the return to `configureModel`. A model template is a base model, whilst a configured model is a base model + extra params.